components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -694,7 +694,10 @@ enum TransactionType {
   ERC20Approve,
   ERC721TransferFrom,
   ERC721SafeTransferFrom,
-  Other
+  Other,
+  SolanaSystemTransfer,  // transfering SOL
+  SolanaSPLTokenTransfer,
+  SolanaSPLTokenTransferWithAssociatedTokenAccountCreation
```

Comment:
It's better to assign values in declaration in case someone adding new fields in the middle, that will break our ToValue and FromValue serialization/deserialization

Reply:
Fixed in https://github.com/brave/brave-core/commit/c9f3ed6e93f80a60f4208e3faedf9a742eb121e4